srcpkgs/lite/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'lite'
+pkgname=lite
+version=1.11
+revision=1
+makedepends="SDL2-devel"
+short_desc="lite is a lightweight text editor written mostly in Lua"
+maintainer="Dakota Richline <drichline@protonmail.com>"
+license="MIT"
+homepage="https://github.com/rxi/lite"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=2fd9466663182da56a36a557d05925d226dc1c5de6fb24e423a7b0056db2eec4
+
+do_build(){
+	bash ${wrksrc}/build.sh
+}
+
+do_install(){
+	vlicense ${wrksrc}/LICENSE
+	vmkdir opt/${pkgname}
```

Comment:
you should be able to use `/usr/share/lite` instead of `/opt` for this (see [AUR](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=lite#n31), for example)